src/main/java/com/puppycrawl/tools/checkstyle/checks/AbstractTypeAwareCheck.java

```diff
@@ -362,11 +364,15 @@ protected final String getCurrentClassName() {
      */
     protected final AbstractClassInfo createClassInfo(final Token name,
                                               final String surroundingClass) {
+        final AbstractClassInfo result;
```

Comment:
I prefer empty `final`s below fully defined `final`s.
It might be because of the length of the lines goes from big to small.
@romani What about you?